CONTRIBUTING.md

```diff
@@ -153,7 +153,6 @@ If you have access to make releases, the process is as follows:
 1. Reset the current major branch to point to the current master `git reset master --hard`
 1. Switch back to master with `git checkout master`
 1. Run tests with `npm test`
-1. Push commits and tags to your fork with `git push && git push --tags`
-1. Push commits and tags upstream with `git push upstream && git push upstream --tags`
+1. Push commits and tags upstream with `git push upstream master && git push upstream --tags`
```

Comment:
Maybe just add a note that they may want to also push to their fork, rather than the extra step?